invenio_rdm_records/marshmallow/fields.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Custom provided fields."""
+from datetime import date
+
+from edtf.parser.grammar import ParseException, level0Expression
+from flask_babelex import lazy_gettext as _
+from marshmallow import fields, missing
+
+
+class EDTFLvl0DateString(fields.Str):
```

Comment:
Comment: Personally I would write "Lvl" out in full instead all the places it's used (i.e. not only in this class name). 

``EDTFLvl0DateString`` -> ``EDTFLevel0DateString`` (or just ``EDTFDateString``)

Reply:
Ah good to know! I was restraining myself here. I'll be a little more liberal with my names.

Reply:
I agree with being more explicit with names in general. I find it easier after to understand the purpose of the class/variable :)